test/clamm.test.ts

```diff
@@ -710,7 +710,7 @@ describe('math tests', () => {
     const minSqrtPriceOutsideDomain = 1n
     const minLiquidity = 1n
     const maxLiquidity = (1n << 256n) - 1n
-    const minOverflowTokenY = 340282366920940n
+    const minOverflowTokenY = 115792089237316195423570985008687907853269984665575031n
```

Comment:
From where the diffrence come from?

Reply:
`SqrtPrice` type on Casper is `u128` and `SqrtPrice` on Alephium is `U256` that's why required number of tokens to cause overflow is higher